src/OpenLoco/Windows/ScenarioSelect.cpp

```diff
@@ -0,0 +1,503 @@
+                return;
+
+            // Delayed start for competing companies
+            args = FormatArguments();
+            args.push<uint16_t>(scenarioInfo->competingCompanyDelay);
+            competitionStringId = scenarioInfo->numCompetingCompanies == 1 ? StringIds::competition_not_starting_for_month : StringIds::competition_not_starting_for_months;
+            Gfx::drawString_494B3F(*dpi, x, y, Colour::black, competitionStringId, &args);
+        }
+    }
+
+    // 0x00443D02
+    static void drawScroll(window* self, Gfx::drawpixelinfo_t* const dpi, uint32_t)
+    {
+        auto colour = Colour::getShade(self->colours[1], 4);
+        Gfx::clearSingle(*dpi, colour);
+
+        using namespace scenariomgr;
+        auto numScenarios = getNumScenariosByCategory(self->current_tab);
+
+        int16_t y = 0;
+        for (auto i = 0; i < numScenarios; i++)
+        {
+            auto* scenarioInfo = getNthScenarioFromCategory(self->current_tab, i);
+            if (scenarioInfo == nullptr)
+                continue;
+
+            if (y + ROW_HEIGHT < dpi->y || y >= dpi->y + dpi->height + ROW_HEIGHT)
+                continue;
+
+            // Highlight selected item
+            auto formatStringId = StringIds::black_stringid;
+            if (scenarioInfo == reinterpret_cast<ScenarioIndexEntry*>(self->info))
+            {
+                Gfx::drawRect(dpi, 0, y, self->width, ROW_HEIGHT - 1, 0x2000000 | 48);
+                formatStringId = StringIds::wcolour2_stringid;
+            }
+
+            // Scenario name
+            {
+                auto str = const_cast<char*>(StringManager::getString(StringIds::buffer_2039));
+                strncpy(str, scenarioInfo->scenarioName, std::size(scenarioInfo->scenarioName));
+
+                auto args = FormatArguments();
+                args.push(StringIds::buffer_2039);
+
+                const int16_t x = 210;
+                Gfx::drawStringCentred(*dpi, x, y + 1, Colour::black, formatStringId, &args);
+            }
+
+            // Completed?
+            if (!scenarioInfo->hasFlag(ScenarioIndexFlags::completed))
+            {
+                y += ROW_HEIGHT;
+                continue;
+            }
+
+            // Draw checkmark to indicate completion
+            Gfx::drawImage(dpi, 395, y + 1, 3629);
+
+            // 'Completed by' info
+            {
+                auto str = const_cast<char*>(StringManager::getString(StringIds::buffer_2039));
+                strncpy(str, scenarioInfo->highscoreName, std::size(scenarioInfo->highscoreName));
+
+                auto args = FormatArguments();
+                args.push(StringIds::completed_by_name_in_years_months);
+                args.push(StringIds::buffer_2039);
+                args.push<uint16_t>(scenarioInfo->completedMonths / 12);
+                args.push<uint16_t>(scenarioInfo->completedMonths % 12);
+
+                const int16_t x = 210;
```

Comment:
Maybe get the list width and divide that by two? Can also use that with the checkmark to right-align it.